fiasco/util/util.py

```diff
@@ -113,8 +113,9 @@ def walk_sub_dir(subdir):
 
         return subdir_files
 
-    non_ion_subdirs = ['abundance', 'ioneq', 'ip', 'continuum', 'dem']
+    non_ion_subdirs = ['abundance', 'ip', 'continuum', 'dem']
```

Comment:
This should not be removed.

Reply:
Let me double check this.  I changed it because it was giving errors relating to missing data, presumably because of the mismatch in `.ioneq` and `ionization_fraction` again.  Certainly possible that I'm just being dumb!

Reply:
Yup, just being dumb.  I swear there were errors, but I can't reproduce them now, probably because I made a mistake somewhere else that is now fixed.

Oh well, thanks for catching it!